sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -875,4 +931,9 @@ public void update(Observable observable) {
             suppressOrRevive();
         }
     }
+
+    @VisibleForTesting
+    void setApiServerDisabled(boolean disabled) {
```

Comment:
(very optional/subjective) alternate option: a `protected` method that returns whether the API server is enabled, and then a `TestDefaultScheduler extends DefaultScheduler` which overrides that protected method